qvm.asd

```diff
@@ -79,4 +79,6 @@
                (:file "stabilizer-qvm")
                (:file "execution")
                (:file "path-simulate")
-               (:file "misc")))
+               (:file "misc")
+               (:file "channel-qvm")
+               (:file "toy-qvm")))
```

Comment:
Looks like `toy-qvm` should be changed to `approximate-qvm.lisp`?

Reply:
yup, the `toy-qvm` was an initial lisp exercise I did with Robert.